src/Symfony/Component/Serializer/Normalizer/NormalizerInterface.php

```diff
@@ -41,11 +43,32 @@ public function normalize(mixed $object, string $format = null, array $context =
     /**
      * Checks whether the given class is supported for normalization by this normalizer.
      *
+     * Since Symfony 6.3, this method will only be called if the $data type is
+     * included in the supported types returned by getSupportedTypes().
+     *
+     * @see getSupportedTypes()
+     *
      * @param mixed       $data    Data to normalize
-     * @param string|null $format The format being (de-)serialized from or into
+     * @param string|null $format  The format being (de-)serialized from or into
      * @param array       $context Context options for the normalizer
      *
      * @return bool
      */
     public function supportsNormalization(mixed $data, string $format = null /* , array $context = [] */);
+
+    /**
+     * Returns the types potentially supported by this normalizer.
+     *
+     * For each supported formats (if applicable), the supported types should be
+     * returned as keys, and each type should be mapped to a boolean indicating
+     * if the result of supportsNormalization() can be cached or not
+     * (a result cannot be cached when it depends on the context or on the data.)
```

Comment:
"(a result cannot be cached when it depends on the context or on the data)." (dot after the parenthesis) ?